layers/meta-resin-allwinner/recipes-kernel/linux/linux-mainline-4.17.3/add-OrangePi-Zero-Wifi.patch

```diff
@@ -0,0 +1,89 @@
+From 9578c534e43ee222971e486ac0e9b6e12cdcdf7f Mon Sep 17 00:00:00 2001
```

Comment:
I see this patch is in the thud branch of meta-sunxi. Can you remove it from here?
We will have thud support shortly, it will be best if we do not duplicate it here also